src/AboutUs/View/page.twig

```diff
@@ -3,7 +3,28 @@
 	<body class="top-wave">
 	{% endblock %}
 	<section class="container">
-		<h1>O nas</h1>
+		<h1 class="u-center-text">O nas</h1>
+		<br>
```

Comment:
h1 and p are block elements. 
br is used to move to next line.
If you want t have space between h1  and p you can add styles: margin bottom to h1 or margin top to p

Reply:
changed, done